spec/DataParser_spec.js

```diff
@@ -1,8 +1,151 @@
 var DataParser = require('../src/DataParser');
 
 describe('Data Parser', function() {
-  it('should fail this test', function() {
-    DataParser();
-    expect(true).toBe(false);
+  it('should return empty object for empty data', function() {
+    var raw_data = new DataParser("");
```

Comment:
I know I told you that we should do `snake_case`, but I think the general JS formatting is to use `camelCase` for variables and functions. My bad.